views/_layout.njk

```diff
@@ -70,27 +68,77 @@
             </ul> <!-- /navbar-nav mr-auto -->
           </div> <!-- /navbarCollapse -->
         </div> <!-- /container-fluid-->
-        </nav>
-        {% endblock %} <!-- /end navbar block -->
+      </nav>
+    {% endblock %} <!-- /end navbar block -->
+
```

Comment:
These alert messages are defined in the base _layout and inherited by every page that "extends" it. The messages are collapsed by default. The functions to hide/show them are in the scripts at the bottom of this page. If messageType exists and is "success" then the alertSuccess element will be shown - which will display the text of the "message" passed in from the route handler. Similar logic applies for messageType "error".